src/app/(proper_react)/redesign/(authenticated)/user/dashboard/fix/high-risk-data-breaches/HighRiskBreachLayout.tsx

```diff
@@ -0,0 +1,165 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+"use client";
+
+import styles from "./HighRiskBreachLayout.module.scss";
+import CreditCardIllustration from "../images/high-risk-data-breach-credit-card.svg";
+import BankAccountIllustration from "../images/high-risk-data-breach-bank-account.svg";
+import PinNumberIllustration from "../images/high-risk-data-breach-pin.svg";
+import SocialSecurityNumberIllustration from "../images/high-risk-data-breach-ssn.svg";
+import Image from "next/image";
+import { UserBreaches } from "../../../../../../../functions/server/getUserBreaches";
+import { usePathname } from "next/navigation";
+import { Button } from "../../../../../../../components/server/Button";
+import Link from "next/link";
+import { useState } from "react";
+
+type HighRiskBreachLayoutProps = {
+  typeOfBreach: "creditCard" | "ssnBreaches" | "bankAccount" | "PIN";
```

Comment:
From one of the files above:
```js
bankAccountBreaches: [ … ],
creditCardNumberBreaches: [ … ],
passwordBreaches: [ … ],
phoneBreaches: [ … ],
pinNumberBreaches: [ … ],
ssnBreaches: [ … ],
```

Does this need to add `"password"` and `"phone"` here (as well as as .svg illustrations)?

Reply:
Also, should this be "PIN" or "pin"?